frontends/election-manager/src/screens/reports_screen.tsx

```diff
@@ -188,10 +189,17 @@ export function ReportsScreen(): JSX.Element {
             <LinkButton primary to={routerPaths.tallyFullReport}>
               {statusPrefix} Full Election Tally Report
             </LinkButton>{' '}
-            {hasConverter && castVoteRecordFiles.wereAdded && (
-              <Button onPress={() => setIsExportResultsModalOpen(true)}>
-                Save Results File
-              </Button>
+            {castVoteRecordFiles.wereAdded && (
+              <React.Fragment>
+                {hasConverter && (
+                  <React.Fragment>
+                    <Button onPress={() => setIsExportResultsModalOpen(true)}>
+                      Save SEMS Results
```

Comment:
Can we make the text here dependent on which converter we're using? SEMS is specific to Mississippi. We have a similar conditional for the converter button.